.env

```diff
@@ -2,6 +2,7 @@ POSTGRES_HOST=database
 POSTGRES_DB=db
 POSTGRES_USER=io-onboarding-pa-api
 POSTGRES_PASSWORD=password
+COOKIE_DOMAIN=.io-onboarding-frontend.com
```

Comment:
Shouldn't be pa-onboarding.io.italia.it?

Reply:
This is just a value to be used as an example, I choosed it in order to explicit that it refers to the frontend domain, even if I explained it in the README file.
In the development environment it will be `.pa-onboarding.dev.io.italia.it`, I already created [a story](https://www.pivotaltracker.com/story/show/168862462) to update the helm chart accordingly.

Reply:
This is just a value to be used as an example, I chose it in order to explicit that it refers to the frontend domain, even if I explained it in the README file.
In the development environment it will be `.pa-onboarding.dev.io.italia.it`, I already created [a story](https://www.pivotaltracker.com/story/show/168862462) to update the helm chart accordingly.